styles/go.dev/catppuccin.user.css

```diff
@@ -0,0 +1,897 @@
+      @svg: escape('<svg width="51" height="4" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M0 4a4 4 0 0 1 4-4h43a4 4 0 0 1 4 4z" fill="@{accent-color}"/></svg>');
+      content: url('data:image/svg+xml,@{svg}');
+    }
+    // /doc/install icon
+    img.CopyPaste-icon {
+      @svg: escape('<svg width="14" height="16" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M10.992 0h-6L4 .992V4H.992L0 4.992V15.04l.992.96H9.04l.96-.96V12h3.12l.88-.96V2.992zm0 1.408 1.584 1.584h-1.584zm-2 13.584h-8v-10H4v6.048l.992.96h4zm4-4h-8v-10H10V4h2.992z" fill="@{text}"/></svg>');
+      content: url('data:image/svg+xml,@{svg}');
+    }
+    // Code Snippet Highlights
+    pre {
+      .comment {
+        color: @overlay2;
+      }
+      ins {
+        color: @blue;
+      }
+    }
+    // Companies in Main Page Contrast
+    & when (@lookup = latte) {
+      .WhoUsesCaseStudyList-caseStudyLink {
+        background-color: @surface0;
+      }
+    }
+  }
+}
+
+@-moz-document domain('pkg.go.dev') {
+  @media (prefers-color-scheme: light) {
+    :root[data-theme='auto'] {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    :root[data-theme='auto']{
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
+  :root[data-theme="dark"] {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root[data-theme="light"] {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
```

Comment:
```suggestion
```

I think it is fine to delete this seeing as it is already included in the first section for `go.dev`.